src/cli/README.md

```diff
@@ -2824,6 +2825,33 @@ Disable radio radio filter.
 Done
 ```
 
+### radiostats
+
+`OPENTHREAD_CONFIG_RADIO_STATS_ENABLE` is required. This feature is only available on FTD and MTD.
+
+The radio statistics shows the time when the radio is in sleep/tx/rx state. The command will show the time of these states since last reset in unit of microseconds. It will also show the percentage of the time.
+
+```bash
+> radiostats
+Radio Statistics:
+TxTime: 14880us, Rx Time: 1491355us, Sleep Time: 20660830us
+Total Time: 22.167s
+Tx Time: 0.014s (0.06%)
```

Comment:
The line 2823 shows the same info with the lines between 2839 and 2841.
 Could we show the values in the format of `%lu.%06lus"` to avoid showing the duplicated messages?